src/nvim/eval/funcs.c

```diff
@@ -6491,6 +6491,26 @@ static void f_prompt_setinterrupt(typval_T *argvars,
     buf->b_prompt_interrupt= interrupt_callback;
 }
 
+/// "prompt_getprompt({buffer})" function
+void f_prompt_getprompt(typval_T *argvars, typval_T *rettv, FunPtr fptr)
+  FUNC_ATTR_NONNULL_ALL
```

Comment:
Weird that this didn't cause a compile error. It failed on my machine last time I tried to add function attribute on eval functions.

Reply:
Some funcs like `f_luaeval()`, `f_reltimestr()` have attribs so I hope things'll be OK. :smile: